spec/frodata/query_spec.rb

```diff
@@ -144,70 +141,16 @@
     end
   end
 
-  describe '#execute' do
-    it { expect(subject).to respond_to(:execute) }
-    it { expect(subject.execute).to be_a(FrOData::Service::Response) }
-  end
-
   describe '#count' do
     it { expect(subject).to respond_to(:count) }
-    it { expect(subject.count).to be_a(Integer) }
-    it { expect(subject.count).to eq(11) }
-
-    context 'with filters' do
-      let(:criteria) { subject[:Name].eq('Bread') }
-
-      it { expect(subject.where(criteria).count).to eq(1) }
-    end
-  end
-
-  describe '#empty?' do
-    it { expect(subject).to respond_to(:empty?) }
-    it { expect(subject.empty?).to eq(false) }
+    it { expect(subject.count).to eq("Products/$count") }
 
-    context 'with filters' do
-      let(:non_empty_criteria) { subject[:Name].eq('Bread') }
-      let(:empty_criteria) { subject[:Name].eq('NonExistent') }
+    # FIXME: Should we support that?
```

Comment:
Probably fine to simplify now and expand support later. Should we track it as an issue?